### DISCUSSION TITLE
Setting for episode's title language

### DISCUSSION THREAD

**Discussion by ySergi** [crunchy-labs/crunchy-cli#414] [Ideas]
As the operating system is in English, the language of the chapter titles is set to English. It would be nice if we could force the language of the title, I would like to be able to have them in Spanish.

**Comment by dok1100** [comment]
have you tried the --lang tag?
crunchy-cli --lang es_ES archive ....
or es_419

unfortunately I don't know which Spanish you prefer.
But it only works if Crunchyroll supports it, else in English

**Reply by ySergi** [reply]
I'm spanish from Spain so I prefer es_ES, the other one if spanish of Latinoamerica.

What is the `archive` thing for? I always use `download`.

**Reply by dok1100** [reply]
There are more options for archives. e.g. Multiple subtitles and/or audio tracks, subtitle fonts can be added, etc

Everything is described in the [Readme](https://github.com/crunchy-labs/crunchy-cli/blob/master/README.md) :)
